core/config/config.go

```diff
@@ -160,6 +160,19 @@ func deleteFromPG(ctx context.Context, db pg.DB) error {
 	return errors.Wrap(err, "deleting config stored in postgres")
 }
 
+// Loads config status from sinkdb to see if other raft nodes have configured
+// so that uncofigured nodes can update.
+func UpdateConfigStatus(ctx context.Context, sdb *sinkdb.DB) (*Config, error) {
+	c := new(Config)
+	ver, err := sdb.Get(ctx, "/core/config", c)
+	if err != nil {
+		return nil, errors.Wrap(err)
+	} else if ver.Exists() {
```

Comment:
Also, can we pass `nil` in as the value (in this case `c`) if we're only checking for the existence of a key?